packages/react-components/src/hooks/useTheming/index.ts

```diff
@@ -0,0 +1 @@
+export * from './useTheming';
```

Comment:
Could you also export the hook and the types from the react-components package?

We want the import to look like:

```
import { useTheme, type UseThemeOptions, type Tokens, type Mode } from '@iot-app-kit/react-components';
```

Reply:
updated